src/server/server_family.cc

```diff
@@ -1860,6 +1863,369 @@ void PrintPrometheusMetrics(uint64_t uptime, const Metrics& m, DflyCmd* dfly_cmd
+    if (added)
+      absl::StrAppend(&resp->body(), type_used_memory_metric);
+  }
+
+  // Stats metrics
+  AppendMetricWithoutLabels("connections_received_total", "", conn_stats.conn_received_cnt,
+                            MetricType::COUNTER, &resp->body());
+
+  AppendMetricHeader("commands_processed_total", "", MetricType::COUNTER, &resp->body());
+  AppendMetricValue("commands_processed_total", conn_stats.command_cnt_main, {"listener"}, {"main"},
+                    &resp->body());
+  AppendMetricValue("commands_processed_total", conn_stats.command_cnt_other, {"listener"},
+                    {"other"}, &resp->body());
+  AppendMetricWithoutLabels("keyspace_hits_total", "", m.events.hits, MetricType::COUNTER,
+                            &resp->body());
+  AppendMetricWithoutLabels("keyspace_misses_total", "", m.events.misses, MetricType::COUNTER,
+                            &resp->body());
+  AppendMetricWithoutLabels("keyspace_mutations_total", "", m.events.mutations, MetricType::COUNTER,
+                            &resp->body());
+  AppendMetricWithoutLabels("lua_interpreter_cnt", "", m.lua_stats.interpreter_cnt,
+                            MetricType::GAUGE, &resp->body());
+  AppendMetricWithoutLabels("freed_memory_lua", "", m.lua_stats.gc_freed_memory,
+                            MetricType::COUNTER, &resp->body());
+  AppendMetricWithoutLabels("lua_blocked_total", "", m.lua_stats.blocked_cnt, MetricType::COUNTER,
+                            &resp->body());
+  AppendMetricWithoutLabels("lua_gc_interpreter_return", "", m.lua_stats.interpreter_return,
+                            MetricType::COUNTER, &resp->body());
+  AppendMetricWithoutLabels("lua_force_gc_calls", "", m.lua_stats.force_gc_calls,
+                            MetricType::COUNTER, &resp->body());
+  AppendMetricWithoutLabels("lua_gc_duration_total_sec", "", m.lua_stats.gc_duration_ns * 1e-9,
+                            MetricType::COUNTER, &resp->body());
+
+  AppendMetricWithoutLabels("backups_total", "", m.loading_stats.backup_count, MetricType::COUNTER,
+                            &resp->body());
+  AppendMetricWithoutLabels("failed_backups_total", "", m.loading_stats.failed_backup_count,
+                            MetricType::COUNTER, &resp->body());
+  AppendMetricWithoutLabels("restores_total", "", m.loading_stats.restore_count,
+                            MetricType::COUNTER, &resp->body());
+  AppendMetricWithoutLabels("failed_restores_total", "", m.loading_stats.failed_restore_count,
+                            MetricType::COUNTER, &resp->body());
+
+  // Net metrics
+  AppendMetricWithoutLabels("net_input_recv_total", "", conn_stats.io_read_cnt, MetricType::COUNTER,
+                            &resp->body());
+  AppendMetricWithoutLabels("net_read_yields_total", "", conn_stats.num_read_yields,
+                            MetricType::COUNTER, &resp->body());
+
+  AppendMetricWithoutLabels("net_input_bytes_total", "", conn_stats.io_read_bytes,
```

Comment:
@romange you did not include this in the list as well as a few more. Shall I also move them under ? dragonfly_memory_by_class_bytes?